experimental/bies/src/lib.rs

```diff
@@ -2,6 +2,47 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+//! The algorithms in this project convert from a BIES matrix (the output of the LSTM segmentation neural network) to concrete segment boundaries.  In BIES, B = beginning of segment; I = inside segment; E = end of segment; and S = single segment (both beginning and end).
+//!
+//! These algorithms always produce valid breakpoint positions (at grapheme cluster boundaries); they don't assume that the neural network always predicts valid positions.
+//!
+//! # Example
+//!
+//! For example, suppose you had the following BIES matrix:
+//!
+//!   |   B   |   I   |   E   |   S   |
+//!   |-------|-------|-------|-------|
+//!   | 0.01  | 0.01  | 0.01  | 0.97  |
+//!   | 0.97  | 0.01  | 0.01  | 0.01  |
+//!   | 0.01  | 0.97  | 0.01  | 0.01  |
+//!   | 0.01  | 0.97  | 0.01  | 0.01  |
+//!   | 0.01  | 0.01  | 0.97  | 0.01  |
+//!   | 0.01  | 0.01  | 0.01  | 0.97  |
+//!   | 0.97  | 0.01  | 0.01  | 0.01  |
+//!   | 0.01  | 0.01  | 0.97  | 0.01  |
+//!
+//! This matrix resolves to:
+//!
+//!   01234567
+//!   SBIIESBE
```

Comment:
Nitpick: Give this 4 spaces of indentation so that it becomes a code block in the README file

Reply:
Thanks! I thought two spaces was sufficient.